qiskit/transpiler/basepasses.py

```diff
@@ -95,6 +95,45 @@ def is_analysis_pass(self):
         """
         return isinstance(self, AnalysisPass)
 
+    def __call__(self, circuit, property_set=None):
+        """Runs the pass on circuit.
+
+        Args:
+            circuit (QuantumCircuit): the dag on which the pass is run.
+            property_set (PropertySet or dict or None): input/output property set.
```

Comment:
Can you point out here that the input property set will be updated in-place by the pass?

Reply:
Done in a256e45ec